[FIX, DOC] Reorganize and improve documentation

tsalo opened the pull request:
<!---
This is a suggested pull request template for tedana.
It's designed to capture information we've found to be useful in reviewing pull requests.

If there is other information that would be helpful to include, please don't hesitate to add it!

Please also label your pull request with the relevant tags.
See here for more information and a list of available options:
https://github.com/ME-ICA/tedana/blob/master/CONTRIBUTING.md#pull-requests
-->

<!-- Please indicate after the # which issue you're closing with this PR.
This is helpful for the maintainers AND will magically close the issue when this
pull request is merged!
https://help.github.com/articles/closing-issues-using-keywords -->
Closes #480 and closes #464.

<!-- Please give a brief overview of what has changed in the PR.
If you're not sure what to write, consider it a note to the maintainers to indicate
what they should be looking for when they review the pull request. -->
Changes proposed in this pull request:

- Link PCA/ICA to respective wikipedia pages (#480).
- Add information about acquisition and processing for quantitative T2* mapping (#464).
- Consolidate acquisition-related sections into one page (**Acquiring multi-echo data**).
- Move considerations from **Considerations** over to **What is multi-echo fMRI?** page and move resources from **Considerations** into new **Resources** page.
- Move **ME-fMRI Parameters & Publications** into new **Acquiring multi-echo data** page.
- Remove "comptable" from **Outputs of tedana** table (we already have the decomposition json in there).
- Fix miscellaneous typos.
- Convert some paragraphs to lists.
- Link to OSF project for protocol PDFs. The actual project is still empty but we should fix that.

handwerkerd commented on docs/acquisition.rst:
```diff
@@ -0,0 +1,160 @@
+
+.. _GE Collaboration Portal: https://collaborate.mr.gehealthcare.com
+
+Other available multi-echo MRI sequences
```
This section is a good addition, but missing a few key points.
1. "While tedana allows the input of a T2* map from any source, and a more accurate T2* map should lead to better results, this hasn't been systematically evaluated yet."
2. There are many ways to calculate quantitative T2* maps. You're listing one way and making some claims that aren't obvious to a reader (i.e. get phase data). This doesn't really tell people where to look for more info. I suggest find your favorite reference for this type of T2* map and reference it here.

tsalo replied:
I'm adding in your first point verbatim, but am a little unsure about what to say for the second. In this section I am basically parroting responses to #464, since I don't have actual experience with quantitative T2* mapping. I can include the paper linked in #464 but I'm not really familiar with any others. My own attempts at searching for T2*-mapping papers have mostly come across methods for scanning knees.

handwerkerd replied:
In that case, here's my suggested re-write of that paragraph. As a general message, I think we should focus on the fact that multi-echo is used in other ways and that some of these ways can calculate potentially useful T2* maps. The original text got specific in ways that left gaps of omission. I've tried to get the main point across without over-specification. (If you like my text, make sure the links are appropriate & work... I'm just drafting here)

"In addition to ME-fMRI, other MR sequences benefit from acquiring multiple
 echoes, including T1-weighted imaging (MEMPRAGE) and susceptibility weighted imaging.
 While most of these kinds of sequences fall outside the purview of this documentation, quantitative T2* mapping is relevant since a baseline T2* map is used in several processing steps including :ref:`optimal combination`.  While the T2* map estimated directly from fMRI time series is noisy, no current study quantifies the benefit to optimal combination or tedana denoising if a higher quality T2* map is used. Some benefit is likely, so, if a T2* map is independently calculated, it can be used as an input to many functions in the tedana workflow.

There are many ways to calculate T2* maps, with some using multi-echo acquisitions. We are not presenting an expansive review of this literature here, but https://doi.org/10.1016/j.ejro.2018.12.006 and https://doi.org/10.1016/j.neuroimage.2012.01.053 are good places to start learning more about this topic."

tsalo replied:
Looks great. Thank you!

handwerkerd replied:
One really minor suggestion to take or leave. When you reference "Cohen-Adad et al. (2012) and Ruuth et al. (2019)" add links to the DOIs for those papers.

tsalo replied:
There are supposed to be links. I must have messed up the hyperlinking, but I'll fix it now.

tsalo replied:
Ah, I added the actual references below but forgot to actually format the links in the text.

handwerkerd commented:
I'm not sure I like "Move ME-fMRI Parameters & Publications into new Acquiring multi-echo data page" The parameters and publications are their own thing and don't really flow well with the rest of the acquiring page. If you don't want that as a stand-alone page, perhaps put them in resources.rst at the bottom of the publications section?

handwerkerd commented on docs/acquisition.rst:
```diff
@@ -0,0 +1,160 @@
+    In order to increase the number of contrasts ("echoes") you may need to first increase the TR, shorten the
+    first TE and/or enable in-plane acceleration.
+    For typically used parameters see the `parameters and publications page`_
+.. _parameters and publications page: https://tedana.readthedocs.io/en/latest/publications.html
```
publications.html is now a dead link. Once there's an agreement where this should be, update the link.

handwerkerd commented on docs/approach.rst:
```diff
@@ -235,23 +236,23 @@ in a dimensionally reduced version of the dataset which is then used in the `TED
+across the echo times.
 
 TE-dependence (:math:`R_2` or :math:`1/T_{2}^*`) and TE-independence (:math:`S_0`) models can then
 be fit to these betas.
```
:ref:`dependence models` does not seem to be working in the compiled file. I know we had a similar issue before & it worked on the actual RTD site, but the other :ref: links seem to be working and this one does not. Not sure if something changed in this revision or an earlier revision, or if it will compile correctly on the RTD site.

tsalo replied:
I can fix this with a hidden toctree and a new parameter in the html configuration in conf.py.

handwerkerd replied:
At least in my local running of <make html> the link to dependence models is still not working.

tsalo replied:
It's probably a Sphinx version issue. The `:hidden:` directive and `:includehidden:` parameter were both added in like 1.4 I think. Tedana's current requirements use a later version than that and the RTD CI build didn't have any warnings (just checked) so if you update your version that might help.

handwerkerd replied:
Not sure what the issue is, but the RTD page that's outputted from the CI  documentation creation that's still pending looks fine so this is resolved.

handwerkerd commented:
Besides fixing broken links, consider all my comments here suggestions. Once the links are fixed, I'll approve the review.

tsalo commented on docs/acquisition.rst:
```diff
@@ -0,0 +1,167 @@
+We have attempted to compile some basic multi-echo fMRI protocols in an `OSF project`_.
+These protocols should not be considered "canonical".
+If you would like to use one of them, please customize it for your own purposes
+and make sure to pilot it before using it in a study.
```
@handwerkerd In fixing the OSF link you noticed, I also realized that we have a ME-fMRI sequences project on the OSF as well (https://osf.io/ebkrp/). I want to link to it (even though we don't actually have anything in there yet), but also want to make sure that the language reflects your past concerns that folks would treat any protocols we provide as "best practice" without critically evaluating and customizing them themselves.

What do you think of the above language?

handwerkerd replied:
How about: "The parameter choices in these protocols run and seem reasonable, but the have not been optimized for a specific situation. They are a good starting point for someone designing a study, but should not be considered canonical. If you would like to use one of them, please customize it for your own purposes and make sure to run pilot scans to test your choices."

tsalo replied:
Added. Thanks!

tsalo commented:
>Perhaps keep the parameter histograms in "acquisition" and move the google doc list of manuscripts into "resources"

That's a great recommendation and the resulting docs make more sense, I think.

tsalo commented:
I think I've addressed all comments, and the RTD build is passing even though the CI doesn't show that. I don't know how to retrigger a build in the hopes that it will send the correct signal back to the CI, so I figured I'd just post a comment with that info.

tsalo commented:
Thank you both for your reviews (especially @handwerkerd)! Merging now.
